### PR TITLE
OSDOCS-8499: RHOSP release notes added

### PR DESCRIPTION
**Pull request by sheriff-rh**
Version(s):
4.15

Issue:
https://issues.redhat.com/browse/OSDOCS-8499

Link to docs preview (VPN required):
[4.15 release notes](https://file.rdu.redhat.com/antaylor/OSDOCS-8499/release_notes/ocp-4-15-release-notes.html) **I recommend `ctrl+f` for "table 29"**

QE review:
- [x] QE has approved this change.

Additional information:
This update adds more information to the tech preview table, as some features are newly GA and some are newly TP. Deprecation of Kuryr has previously been documented elsewhere in the release notes, as well as CAPO description as a new TP feature, which I added to the TP table as well. Control Plane with rootVolumes and etcd on local disk was mentioned in the 4.14 release notes, however I added it to the TP table too for this release. 

Please let me know if I missed anything!

### CODE REVIEW

**Review by EmilienM — APPROVED** [review]
/lgtm

**Review by mletalie — COMMENTED** [review]
/remove-label peer-review-needed
/remove-label peer-review-in-progress
/label peer-review-done
/lgtm

File: release_notes/ocp-4-15-release-notes.adoc [hunk 1]
```diff
@@ -1293,6 +1293,21 @@ In the following tables, features are marked with the following statuses:
 |Dual-stack networking with installer-provisioned infrastructure
 |Not Available
 |Technology Preview
+|General Availability
+
+|Dual-stack networking with user-provisioned infrastructure
+|Not Available
+|Technology Preview
```
Comment (MaysaMacedo) [inline release_notes/ocp-4-15-release-notes.adoc:1300]: UPI was not available as TP in 4.14 only IPI was.
UPI work was developed on 4.15 and is moving straight to GA on 4.15. Can you update that? thanks

**Review by MaysaMacedo — CHANGES_REQUESTED** [review]
thanks for the release notes! There is just a detail:

**Comment by MaysaMacedo** [comment]
/hold
to give a chance for my comment to be fixed.
Feel free to unhlod when you checked it

File: release_notes/ocp-4-15-release-notes.adoc (hunk 1 above)
Comment (sheriff-rh) [reply release_notes/ocp-4-15-release-notes.adoc]: Updated - thanks @MaysaMacedo !

File: release_notes/ocp-4-15-release-notes.adoc [hunk 2]
```diff
@@ -1303,6 +1303,21 @@ In the following tables, features are marked with the following statuses:
...
+
+|Dual-stack networking with user-provisioned infrastructure
+|Not Available
+|Not Available
+|General Availability
+
+|Cluster API Provider for OpenStack (CAPO)
+|Not Available
+|Not Available
+|Technology Preview
```
[the remark this answers is not in the document]
Comment (dulek) [reply release_notes/ocp-4-15-release-notes.adoc:1316]: And while we're at it - I don't think CAPI stands for "Coherent Accelerator Processor Interface", but for "Cluster API" and MAPO is not "Mail Application Programming Interface" but "Machine API". Can you please fix that in that note too?

[the remark this answers is not in the document]
Comment (sheriff-rh) [reply release_notes/ocp-4-15-release-notes.adoc:1316]: @dulek do you mean to name it `CAPO integration into the cluster CAPI Operator` ? 

I'll see if I can link to the paragraph too with a footnote or hyperlink, that's a good idea.

[the remark this answers is not in the document]
Comment (sheriff-rh) [reply release_notes/ocp-4-15-release-notes.adoc]: I applied all of the changes @dulek, thanks for the good ideas. The footnote links directly to the paragraph.
image

**Comment by eurijon** [comment]
hey, it lgtm with the latest changes too

**Comment by sheriff-rh** [comment]
Thanks Jon! Proceeding with peer review and then merging.

**Pull request merged**